hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/audit/parser/TestAuditParser.java

```diff
@@ -42,7 +42,7 @@
 import java.util.List;
 
 /**
- * Tests GenerateOzoneRequiredConfigurations.
+ * Tests TestAuditParser.
```

Comment:
It should be AuditParser.

Reply:
@bharatviswa504 Thanks for your review 
I already updated my patch